test/test_monge_ampere.py

```diff
@@ -64,19 +75,20 @@ def test_continue(method, exports=False):
     #        for the relaxation method, which is concerning.
 
 
-def test_change_monitor(method, exports=False):
```

Comment:
Did you turn exports on for debugging? I think we should have them off by default so they don't get used in the CI.

Reply:
Ah yeah. Thanks, good catch